testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -8062,146 +8062,158 @@ void checkRegexpExtract(SqlOperatorFixture f0, FunctionAlias functionAlias) {
     f.checkNull("array_concat(cast(null as integer array), array[1])");
   }
 
-  /** Tests {@code ARRAY_CONTAINS} function from Spark. */
+  /** Tests {@code ARRAY_CONTAINS} function from Spark、Hive. */
```

Comment:
Not the usual comma that we normally use, please change it (8 occurrences in this file) (`、` => `,` )

Reply:
Done

Reply:
Sorry but the change is incomplete: you modified one ocurrence, but there are still 7 more occurrences of this character that your PR has introduced throughout this file, could you please correct them?

Reply:
Done，Thanks @rubenada